README.md

```diff
@@ -2,41 +2,61 @@
+technology stack extended with our DID, CType, Attestation and hierarchical Trust Modules.
+Substrate Documentation:
 
-## Table of Contents
-- [How to use TLDR](#how-to-use-tldr)
+[High Level Docs](https://substrate.dev/docs/en/getting-started/)
+[JSON-RPC](https://polkadot.js.org/api/substrate/rpc.html)
+[Reference Rust Docs](https://substrate.dev/rustdocs/v1.0/substrate_service/index.html)
+
+- [How to use TL;DR](#how-to-use-tl-dr)
 - [How to use](#how-to-use)
-  - [Building / Images](#building--images)
-    - [Dockerhub](#dockerhub)
-    - [Building docker image](#building-docker-image)
-    - [Build code without docker](#build-code-without-docker)
-  - [Commands](#commands)
-    - [Helper script](#helper-script)
-    - [Node binary](#node-binary)
-  - [Examples](#examples)
-    - [Running a local node that connects to KILT prototype testnet in AWS](#running-a-local-node-that-connects-to-kilt-prototype-testnet-in-aws)
-    - [Running a node with local image, which runs a dev-chain](#running-a-node-with-local-image-which-runs-a-dev-chain)
+  * [Images / Building](#images---building)
+    + [Dockerhub](#dockerhub)
+    + [Building docker image](#building-docker-image)
+    + [Build code without docker](#build-code-without-docker)
+    + [Building in dev mode](#building-in-dev-mode)
+    + [Building in performant release mode](#building-in-performant-release-mode)
+  * [Commands](#commands)
+    + [Helper script](#helper-script)
+    + [Node binary](#node-binary)
+  * [Examples](#examples)
+    + [Running a local node that connects to KILT prototype testnet in AWS](#running-a-local-node-that-connects-to-kilt-prototype-testnet-in-aws)
+    + [Running a node with local image, which runs a dev-chain](#running-a-node-with-local-image--which-runs-a-dev-chain)
 - [Development with AWS images](#development-with-aws-images)
-- [Updating with latest substrate-node-template](#updating-with-latest-substrate-node-template)
-
-## How to use TLDR
+- [Node Modules functionalities](#node-modules-functionalities)
+  * [DID Module](#did-module)
+    + [Add](#add)
+    + [CRUD](#crud)
+  * [CTYPE Module](#ctype-module)
+  * [Attestation Module](#attestation-module)
+    + [Add](#add)
+    + [Revoke](#revoke)
+    + [Lookup](#lookup)
+  * [Hierarchy of Trust Module](#hierarchy-of-trust-module)
+    + [Create root](#create-root)
+    + [Add delegation](#add-delegation)
+    + [Revoke](#revoke-1)
+- [Updating to latest substrate-node-template](#updating-to-latest-substrate-node-template)
+
+## How to use TL;DR
 Start chain and connect to alice bootnode:
 ```
 docker run -p 9944:9944 kiltprotocol/mashnet-node ./start-node.sh --connect-to alice
 ```
 
-Start dev chain for local development
+Start dev chain (producing blocks without requirements) for local development with all (default is local) WebSocket Interfaces and Remote Procedure Calls enabled and specified WebSockets RPC server TCP port
```

Comment:
Great to see these extra details. 
One thing: "default is local": Do you mean that by default the chain starts in local dev mode? Maybe this could be made a bit clearer.
--> Suggestion: detail "(default is local)" + maybe move it to a separate sentence for brevity.

Reply:
Yes good suggestion